docs/examples/use_cases/tensorflow/resnet-n/nvutils/image_processing.py

```diff
@@ -15,170 +14,316 @@
+                        index_path=tfrec_idx_filenames,
+                        random_shuffle=True,
+                        shard_id=shard_id,
+                        num_shards=num_gpus,
+                        initial_fill=10000,
+                        features={
+                            'image/encoded': tfrec.FixedLenFeature((), tfrec.string, ""),
+                            'image/class/label': tfrec.FixedLenFeature([1], tfrec.int64,  -1),
+                            'image/class/text': tfrec.FixedLenFeature([ ], tfrec.string, ''),
+                            'image/object/bbox/xmin': tfrec.VarLenFeature(tfrec.float32, 0.0),
+                            'image/object/bbox/ymin': tfrec.VarLenFeature(tfrec.float32, 0.0),
+                            'image/object/bbox/xmax': tfrec.VarLenFeature(tfrec.float32, 0.0),
+                            'image/object/bbox/ymax': tfrec.VarLenFeature(tfrec.float32, 0.0)})
+
+        decode_device = "cpu" if dali_cpu else "mixed"
+        resize_device = "cpu" if dali_cpu else "gpu"
+        if training:
+            images = fn.image_decoder_random_crop(
+                inputs["image/encoded"],
+                device=decode_device,
+                output_type=types.RGB,
+                random_aspect_ratio=[0.75, 1.25],
+                random_area=[0.05, 1.0],
+                num_attempts=100)
+            images = fn.resize(images, device=resize_device, resize_x=width,
+                                        resize_y=height)
         else:
-            if dali_cpu:
-                self.decode = dali.ops.ImageDecoder(
-                    device="cpu",
-                    output_type=dali.types.RGB)
-                resize_device = "cpu"
-            else:
-                self.decode = dali.ops.ImageDecoder(
-                    device="mixed",
-                    output_type=dali.types.RGB)
-                resize_device = "gpu"
+            images = fn.image_decoder(
+                inputs["image/encoded"],
+                device=decode_device,
+                output_type=types.RGB)
             # Make sure that every image > 224 for CropMirrorNormalize
-            self.resize = dali.ops.Resize (device=resize_device, resize_shorter=256)
+            images = fn.resize(images, device=resize_device, resize_shorter=256)
 
-        self.normalize = dali.ops.CropMirrorNormalize(
+        images = fn.crop_mirror_normalize(
+            images,
             device="gpu",
-            dtype=dali.types.FLOAT,
+            dtype=types.FLOAT,
             crop=(height, width),
             mean=[123.68, 116.78, 103.94],
-            output_layout="HWC")
-        self.mirror = dali.ops.CoinFlip()
-        self.iter = 0
-
-    def define_graph(self):
-        # Read images and labels
-        inputs = self.input(name="Reader")
-        images = inputs["image/encoded"]
+            std=[58.4, 57.12, 57.3],
+            output_layout=types.NHWC,
```

Comment:
```suggestion
            output_layout="HWC",
```
Using layouts in `types.*` is deprecated

Reply:
Copy-paste. Fixed